client/src/VideoItem.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const VideoItem = ({ video, onDelete, onVote }) => {
+
+  const videoId = video.url.split('v=')[1];
```

Comment:
What would happen if the video URL didn't contain `v=`? Can you think of a way of handling that more smoothly?